pkg/cmd/chains/chains.go

```diff
@@ -0,0 +1,134 @@
+package chains
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/tektoncd/chains/pkg/chains/formats"
+	"github.com/tektoncd/chains/pkg/chains/storage"
+	"github.com/tektoncd/chains/pkg/config"
+	"github.com/tektoncd/cli/pkg/cli"
+	"github.com/tektoncd/cli/pkg/flags"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	Namespace   string = "tekton-chains"
```

Comment:
This hardcoding makes it bit worrying as the namespace can be `tekton-pipelines` and `openshift-pipelines` if installed through operator

Reply:
Even for chains? So maybe we should use the `namespace` flag from the CLI and default to "tekton-chains" if it is not supplied?

Reply:
Is there any downside of installing chains in other namespaces? We are currently testing it in operator

Reply:
Where can one see the progress of adding chains to the tekton operator? Also is there any details on when it would be added to OpenShift pipelines or nothing yet?

Reply:
This is the PR on chains https://github.com/tektoncd/operator/pull/630 It is targeted for upcoming releases i.e. Tekton Operator v0.55 and OpenShift Pipelines v1.7.0

Reply:
thanks